config/core/deployments/controller.yaml

```diff
@@ -56,8 +56,6 @@ spec:
           # PingSource
           - name: PING_IMAGE
             value: ko://knative.dev/eventing/cmd/ping/adapter
-          - name: JOB_RUNNER_IMAGE
-            value: ko://knative.dev/eventing/cmd/ping/jobrunner
```

Comment:
This thing is now moved to the manifest and created by the operator.
No need to pass this around.